minard/templates/shifter_information.html

```diff
@@ -0,0 +1,62 @@
+{% from "_formhelpers.html" import render_field_shifter %}
+{% extends "layout.html" %}
+{% block title %} Shifter Information  {% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block body %}
+  {{ super() }}
+
+<div class="container">
+  <div class="row">
+    <h1 align="center">Shifter Information</h1> <br>
+    <table class="table table-hover">
+      <tr> 
+        <th> <h3> <font color='green'> {{ shifter }} </font> </h3> </th>
+        <th> <h3> <font color='green'> {{ updates }} </font> </h3> </th>
+        <th> <h3> <font color='green'> {{ expert }} </font> </h3> <br> </th>
+      </tr>
+    </table>
+    <h3> If you are the current shifter, please fill out the below information at the start of your shift. If you are not the current shifter, do not fill in this form. After hitting Submit, make sure the current shifter and expert information about update properly. If multiple people are on shift, the shift leader should fill this out. This information will only be used for email and text alarm alerts. </h3> <br>
+    <form action="shifter_information" method="POST" role="form">
+      <table class="table table-hover">
+         <tr> 
+           <th> <h4> First name: </h4> </th>
+           <th> <h4> {{ render_field_shifter(form.firstname) }} </h4> </th>
+           <th> Required </th>
+         </tr>
+         <tr>
+           <th> <h4> Last name: </h4> </th>
+           <th> <h4> {{ render_field_shifter(form.lastname) }} </h4> </th>
+           <th> Required </th>
+         </tr>
+         <tr>
+           <th> <h4> Expert: </h4> </th>
+           <th> <h4> {{ form.expert }} </h4> </th>
+           <th> Required, current on-call expert. </th> 
+         </tr>
+         <tr>
+           <th> <h4> Email: </h4> </th>
+           <th> <h4> {{ render_field_shifter(form.email) }} </h4> </th>
+           <th> Optional, leave empty to not receive email alerts.<br>
+                Use full address (eg, username@aol.com) </th>
+         </tr>
+         <tr>
+           <th> <h4> Phone number: </h4> </th>
+           <th> <h4> {{ render_field_shifter(form.phone) }} </h4> </th>
+           <th> Optional, leave empty to not receive phone alerts. <br> 
+                Don't use spaces and include the international code (eg, 15559998888) </th>
+         </tr>
+         <tr>
+           <th> <h4> Country: </th>
+           <th> <h4> {{ render_field_shifter(form.country) }} </h4> </th>
+           <th> Optional, leave empty to not receive phone alerts. <br>
+                This is the country associated with your phone number. <br> 
+                Current valid countries: USA, Canada, Portugal, Germany, UK </th>
+         </tr>
+      </table>
```

Comment:
I think it would be kind of nice to try to keep things consistent across the monitoring site. Here is a suggestion (I'll post a pic of what it looks like and you can decide if you think it looks OK):

```html
<table class="table">
<tr>
    <td> {{ render_field(form.firstname) }} </td>
    <td> Required </td>
</tr>
<tr>
    <td> {{ render_field(form.lastname) }} </td>
    <td> Required
</tr>
```

etc. To make this look nice you'll have to update the name of your labels to be readable, i.e.

```python
class ShifterInfoForm(form):
    firstname = StringField("First Name", ...)
```

I would prefer if you just use the regular `render_field()` here instead of a new one. If you want the help text to be more obvious, which I think is a good idea, perhaps you could update the error part to be:

```
    {% if field.errors %}
        <ul class=errors>
            {% for error in field.errors %}
                <li><div class="alert alert-error">{{ error }}</div></li>
            {% endfor %}
            </ul>
{% endif %}

Reply:
This looks good to me.